blocks/blog-list/blog-list.js

```diff
@@ -23,15 +23,24 @@ export async function renderFilterCard(post) {
   const cardArrow = span({ class: 'card-arrow' });
   cardArrow.innerHTML = await arrowSvg;
 
-  cardText.append(
-    span({ class: 'card-date' }, publicationDate),
-    div({ class: 'card-cta' },
-      a({ class: 'cta-readmore', href: post.path, 'aria-label': placeholders.readMore },
-        placeholders.readMore,
-        cardArrow,
+  if (customerStory) {
+    cardText.append(
+      span({ class: 'card-date' }, publicationDate),
+      div({ class: 'card-cta' },
+        span({ class: 'card-description' }, post.description),
       ),
-    ),
-  );
+    );
+  } else {
+    cardText.append(
+      span({ class: 'card-date' }, publicationDate),
+      div({ class: 'card-cta' },
+        a({ class: 'cta-readmore', href: post.path, 'aria-label': placeholders.readMore },
+          placeholders.readMore,
+          cardArrow,
+        ),
+      ),
+    );
```

Comment:
You don't need to duplicate the code here to add a condition.
You can do something like
```javascript
  cardText.append(
    span({ class: 'card-date' }, publicationDate),
    showDescription
      ? div({ class: 'card-cta' },
        a({ class: 'cta-readmore', href: post.path, 'aria-label': placeholders.readMore },
          placeholders.readMore,
          cardArrow,
        ),
      )
      : span({ class: 'card-description' }, post.description),
  );
```